Cargo.toml

```diff
@@ -17,8 +17,8 @@ log = "0.4.13"
 serde = { version = "1.0.119", features = ["derive"] }
 serde_json = "1.0.61"
 node-bindgen = "4.1.0"
-fluvio = { version = "0.5.0-beta.1", features = ["admin"] }
+fluvio = { git = "https://github.com/infinyon/fluvio", rev = "b7b3d2c16880ea21f1c67246ce9765368c991ebe", features = ["admin"] }
```

Comment:
If you feel like changing this to `version = 0.6.0` now we can but I don't feel strongly about it.